index.d.ts

```diff
@@ -201,47 +202,71 @@ declare const ky: {
 	@param url - `Request` object, `URL` object, or URL string.
 	@returns A promise with `Body` methods added.
 	*/
-	get(url: Input, options?: OptionsWithoutBody): ResponsePromise;
+	get(url: Input, options?: Options): ResponsePromise;
+
+	/**
+	Fetch the given `url` using the option `{method: 'head'}`.
+
+	@param url - `Request` object, `URL` object, or URL string.
+	@returns A promise with `Body` methods added.
+	*/
+	head(url: Input, options?: Options): ResponsePromise;
 
 	/**
 	Fetch the given `url` using the option `{method: 'post'}`.
 
 	@param url - `Request` object, `URL` object, or URL string.
 	@returns A promise with `Body` methods added.
 	*/
-	post(url: Input, options?: OptionsWithBody): ResponsePromise;
+	post(url: Input, options?: Options): ResponsePromise;
 
 	/**
 	Fetch the given `url` using the option `{method: 'put'}`.
 
 	@param url - `Request` object, `URL` object, or URL string.
 	@returns A promise with `Body` methods added.
 	*/
-	put(url: Input, options?: OptionsWithBody): ResponsePromise;
+	put(url: Input, options?: Options): ResponsePromise;
 
 	/**
-	Fetch the given `url` using the option `{method: 'patch'}`.
+	Fetch the given `url` using the option `{method: 'delete'}`.
 
 	@param url - `Request` object, `URL` object, or URL string.
 	@returns A promise with `Body` methods added.
 	*/
-	patch(url: Input, options?: OptionsWithBody): ResponsePromise;
+	delete(url: Input, options?: Options): ResponsePromise;
 
 	/**
-	Fetch the given `url` using the option `{method: 'head'}`.
+	Fetch the given `url` using the option `{method: 'connect'}`.
+
+	@param url - `Request` object, `URL` object, or URL string.
+	@returns A promise with `Body` methods added.
+	*/
+	connect(url: Input, options?: Options): ResponsePromise;
```

Comment:
I don't want methods for `connect`, `options`, `trace`. They are not common.

Reply:
Ha! Was just removing them. The fetch spec says `forbidden methods` as well, though they are in the HTTP spec.